mdt_dynamic_inventory.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python
+
+# (c) 2016, Julian Barnett <jbarnett@tableau.com>
+#
+# This file is part of Ansible.
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+'''
+MDT external inventory script
+=================================
+author: J Barnett 06/23/2016 01:15
+maintainer: J Barnett (github @jbarnett1981)
+'''
+
+import argparse
```

Comment:
Please use different PR for this change.

Reply:
Yea i'm not sure why this is showing up in this PR. I'll delete it and try again